lib/generators/curly/scaffold/scaffold_generator.rb

```diff
@@ -0,0 +1,47 @@
+require "rails/generators/resource_helpers"
+require "generators/curly"
+
+module Curly # :nodoc:
+  module Generators # :nodoc:
+    class ScaffoldGenerator < Base # :nodoc:
+      include Rails::Generators::ResourceHelpers
+
+      source_root File.expand_path("../templates", __FILE__)
+
+      argument :attributes, type: :array, default: [], banner: "field:type field:type"
+
+      def create_root_folder
+        empty_directory File.join("app/views", controller_file_path)
+        empty_directory File.join("app/presenters", controller_file_path)
+      end
+
+      def copy_view_files
+        # Presenter for the singular version of th emodel.
+        template "model_presenter.rb.erb", File.join("app/presenters", controller_file_path, "#{singular_table_name}_presenter.rb")
+        # Presenter with inheritable methods useful in many cases.
+        template "presenter_base.rb.erb", File.join("app/presenters", controller_file_path, "presenter_base.rb")
```

Comment:
This is slightly too opinionated – I feel that people should extract shared methods to modules or base classes when it's needed only.